rocketchat.sls

```diff
@@ -0,0 +1,14 @@
+rocketchat:
+{% for version in ['2.10.1',
+                   '2.9.0'] %}
+  '{{ version }}':
+    {% set PROGRAM_FILES = "%ProgramFiles%" %}
```

Comment:
Put this at the top of the file (outside the for loop) so that it isn't defined during each iteration of the loop